tests/test_pipeline.py

```diff
@@ -25,57 +25,9 @@
 import textwrap
 import unittest
 
-import lsst.pex.config as pexConfig
 import lsst.utils.tests
-from lsst.pipe.base import (
-    Pipeline,
-    PipelineDatasetTypes,
-    PipelineTask,
-    PipelineTaskConfig,
-    PipelineTaskConnections,
-    Struct,
-    TaskDef,
-    connectionTypes,
-)
-from lsst.pipe.base.tests.simpleQGraph import makeSimplePipeline
-
-
-class DummyAddConnections(PipelineTaskConnections, dimensions=()):
-    dummyOutput = connectionTypes.Output("addOutput", "dummyStorage", "add output")
-
-
-class DummyMultiplyConnections(PipelineTaskConnections, dimensions=()):
-    dummyInput = connectionTypes.Input("addOutput", "dummyStorage", "add output")
-
-
-class AddConfig(PipelineTaskConfig, pipelineConnections=DummyAddConnections):
-    addend = pexConfig.Field(doc="amount to add", dtype=float, default=3.1)
-
-
-class AddTask(PipelineTask):
-    ConfigClass = AddConfig
-    _DefaultName = "add"
-
-    def run(self, val):
-        self.metadata.add("add", self.config.addend)
-        return Struct(
-            val=val + self.config.addend,
-        )
-
-
-class MultConfig(PipelineTaskConfig, pipelineConnections=DummyMultiplyConnections):
-    multiplicand = pexConfig.Field(doc="amount by which to multiply", dtype=float, default=2.5)
-
-
-class MultTask(PipelineTask):
-    ConfigClass = MultConfig
-    _DefaultName = "mult"
-
-    def run(self, val):
-        self.metadata.add("mult", self.config.multiplicand)
-        return Struct(
-            val=val * self.config.multiplicand,
-        )
+from lsst.pipe.base import Pipeline, PipelineDatasetTypes, TaskDef
+from lsst.pipe.base.tests.simpleQGraph import AddTask, makeSimplePipeline
```

Comment:
I think most of this duplication extends from before this code lived in pipe_base, and was in ctrl_mpexec, and so the dependency order meant that it could not be imported.